src/app/shared/model/creative.model.spec.ts

```diff
@@ -0,0 +1,23 @@
+import { Observable } from 'rxjs/Observable';
+import { MockHalDoc } from 'ngx-prx-styleguide';
+import { CreativeModel } from './creative.model';
+
+describe('CreativeModel', () => {
+
+  const makeCreative = (data?: any) => {
+    const campaignDoc = new MockHalDoc({copy: 'test-campaign'});
+    const creativeDoc = new MockHalDoc(data);
+    return new CreativeModel(campaignDoc, creativeDoc);
+  };
+
+  it('loads data from the haldoc', () => {
+    const creative = makeCreative({filename: 'creative-one'});
+    expect(creative.filename).toEqual('creative-one');
+    expect(creative.isNew).toBeFalsy();
```

Comment:
Kinda just "thinking out loud" on this comment... I was a bit confused about isNew being Falsey for a minute because I was thinking it needed a save, but yeah it looks like when you don't give it any data, init sets isNew to true, but when you do give it data in the constructor, isNew is false.

Reply:
copied most of the tests from publish -- happy to alter if you think that would help for clarity!